From a2e216a6fe1f07870727557e03089c531e488172 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Tue, 12 Jul 2022 13:58:03 +0300
Subject: [PATCH] [SW-94864] rom: export dma dest api

Export dma dest api from rom in order to re-use in ppboot.

Change-Id: Ibcd20ec6b8f0f7019b61123fa00c589225ca9a3a
---

diff --git a/arch/gaudi3/arc/arcboot_rom.txt b/arch/gaudi3/arc/arcboot_rom.txt
index 2407381..c5bafc7 100644
--- a/arch/gaudi3/arc/arcboot_rom.txt
+++ b/arch/gaudi3/arc/arcboot_rom.txt
@@ -7,3 +7,4 @@
 dma_copy_image_v2
 security_rom_get_hash
 arcboot_rom_mem_init
+arcboot_rom_get_dma_dest_type
